AsyncDisplayKit/Layout/ASLayoutElement.h

```diff
@@ -8,7 +8,7 @@
 //  of patent rights can be found in the PATENTS file in the same directory.
 //
 
-#import <AsyncDisplayKit/ASDimension.h>
+#import <AsyncDisplayKit/ASDimensionInternal.h>
```

Comment:
Similar as the Umbrella, if it is true that this file can be imported externally (which I think that it can) -- then ideally, the *Internal.h would be imported only in the .m/.mm file.